core/control-plane/control-plane-aggregate-services/src/main/java/org/eclipse/edc/connector/service/transferprocess/TransferProcessServiceImpl.java

```diff
@@ -91,6 +92,11 @@ public ServiceResult<Stream<TransferProcess>> query(QuerySpec query) {
         });
     }
 
+    @Override
+    public ServiceResult<Void> stop(StopTransferCommand command) {
```

Comment:
to keep consistency with the other methods, this should probably also be `@NotNull`